python/feathub/processors/flink/table_builder/tests/test_flink_processor.py

```diff
@@ -445,27 +444,6 @@ def get_client(self, extra_config: Optional[Dict] = None) -> FeathubClient:
             )
         return self._cached_clients[str(extra_config)]
 
-    def test_read_write(self):
-        source = self.create_file_source(self.input_data)
-
-        sink_path = tempfile.NamedTemporaryFile(dir=self.temp_dir, suffix=".csv").name
-
-        sink = FileSystemSink(sink_path, "csv")
-
-        self.client.materialize_features(
-            features=source,
-            sink=sink,
-            allow_overwrite=True,
-        ).wait()
-
-        files = glob.glob(f"{sink_path}/*")
-        df = pd.DataFrame()
-        for f in files:
-            csv = pd.read_csv(f, names=["name", "cost", "distance", "time"])
-            df = df.append(csv)
-        df = df.sort_values(by=["time"]).reset_index(drop=True)
-        self.assertTrue(self.input_data.equals(df))
-
     def test_unsupported_file_format(self):
```

Comment:
Should we also verify this test case against LocalProcessor and SparkProcessor, and do you think we should add support for remote FS for LocalProcessor in this PR?

Reply:
I think the behavior in this test is specific to the FlinkProcessor. 

Given that LocalProcessor is only used for local development and testing, I am not sure if we should support remote FS for LocalProcessor.  Anyway, I think supporting remote FS for LocalProcessor is out of the scope of this PR.